addons/binding/org.openhab.binding.allplay/OSGI-INF/.gitignore

```diff
@@ -0,0 +1 @@
+/*.xml
```

Comment:
we have to add it in every OSGI-INF directory until we have converted ALL OSGI services to the DS annotations, otherwise we do not have those xml files under version control that are created manually.